airflow/api/client/api_client.py

```diff
@@ -60,12 +60,13 @@ def get_pools(self):
         """Get all pools."""
         raise NotImplementedError()
 
-    def create_pool(self, name, slots, description):
+    def create_pool(self, name, slots, description, include_deferred):
```

Comment:
Not entirely sure, I think this needs a default `False` for backward compatibility.

Reply:
This is old, deprecated, experimental API client that we don't use for years. As such I'd argue that we just should not change it at all.

Reply:
Can you remove those changes from "airflow/api/client" packages @Usiel please? I am eyeing to add that change to 2.7.0 if possible, so if there is a way to do it quickly that would be fantastic.

Reply:
I took the liberty to add a fixup to remove the "experimental api" changes and rebase the change on top of latest main. I will try to convince the release manager to include that one in 2.7.0 - it would be a pity to wait till 2.8 for that one as there is clearly a need.

Reply:
> I will try to convince the release manager to include that one in 2.7.0 - it would be a pity to wait till 2.8 for that one as there is clearly a need.

Thank you, that'd be great!

Unfortunately the `pool_command` makes use of `airflow.api.client`. So if we don't want any changes to the API client, we'd also have to undo the changes for the CLI pool commands (specifically the `airflow pool set ...` cmd). I can do that now, if that's what we're going for.

Reply:
Yeah. just realied taht. My bad. Actually - this was fine how it was before. I realized that the CLI command is still using tha API (Historical reason) but it is NOT the experimental API- it was internal API used in the past by the clients and we moved away from it in newer clients. There is absolutely no guarantees about this API that we give - so your change was good as it was. 

resetting my change.